src/libstd/rt/util.rs

```diff
@@ -144,56 +144,6 @@ pub fn abort(args: fmt::Arguments) -> ! {
     let _ = write!(&mut w, "{}", args);
     let msg = str::from_utf8(&w.buf[0..w.pos]).unwrap_or("aborted");
     let msg = if msg.is_empty() {"aborted"} else {msg};
```

Comment:
Sorry, I meant the whole writing to a buffer thing. That was only done for the fortune hash. Can just write the format args to stderr now.